README.markdown

```diff
@@ -241,6 +242,13 @@ API Implemented
 
 [Back to TOC](#table-of-contents)
 
+## resty.core.socket_tcp
```

Comment:
I don't quite like this module name. How about `resty.core.socket.tcp`?

Reply:
Yes, we can rename the Lua file.

Reply:
Renamed.